embark/uploader/models.py

```diff
@@ -153,7 +153,7 @@ class Firmware
     cwe_checker = BooleanFieldExpertMode(
         help_text='Enables cwe-checker,-c will be added', default=False, expert_mode=True, blank=True)
     dev_mode = BooleanFieldExpertMode(
-        help_text='Run emba in developer mode, -D will be added (disabling developer mode is currently not supported)', default=True, expert_mode=True, blank=True)
+        help_text='Run emba in developer mode, -D will be added ', default=True, expert_mode=True, blank=True)
```

Comment:
after landing this PR we need to remove this option completely.

Reply:
default set to False now